models/DataObject/Data/BlockElement.php

```diff
@@ -154,6 +155,18 @@ function ($currentValue) {
             ),
             new UnmarshalMatcher()
         );
+
+        $copier->addFilter(new \DeepCopy\Filter\KeepFilter(), new class() implements \DeepCopy\Matcher\Matcher {
+            public function matches($object, $property)
+            {
+                if ($object instanceof AbstractElement) {
+                    return true;
+                }
+
+                return false;
```

Comment:
Could be simplified to:

```suggestion
                return $object instanceof AbstractElement;
```